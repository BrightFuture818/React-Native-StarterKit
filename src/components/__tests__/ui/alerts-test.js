/**
 * Test to check if the component renders correctly
 */
/* global it expect */

import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import Alerts from '@ui/Alerts';

/**
 * Check if alerts renders correctly
 * and asserting it to the matching snapshot
 */
it('Alerts renders correctly', () => {
  const tree = renderer.create(
    <Alerts />,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
