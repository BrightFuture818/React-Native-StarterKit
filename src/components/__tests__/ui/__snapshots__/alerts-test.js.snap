exports[`test Alerts renders correctly 1`] = `
<View
  style={
    Object {
      "left": 0,
      "right": 0,
    }
  } />
`;
